packages/deployer/internal/ast/ast-helper.js

```diff
@@ -12,6 +12,52 @@ function findContractNode(contractName, nodeOrAst) {
   return null;
 }
 
+function findContractNodeWithId(contractId, contracts) {
+  for (var [contractName, contractAST] of Object.entries(contracts)) {
+    const contractNode = findContractNode(contractName, contractAST);
+
+    if (contractNode.id === contractId) {
+      return contractNode;
+    }
+  }
+
+  return null;
+}
+
+function findDependenciesOf(contractName, contracts) {
+  const contractNode = findContractNode(contractName, contracts[contractName]);
+  if (!contractNode) {
+    return null;
+  }
+
+  let dependencies = [];
+
+  contractNode.linearizedBaseContracts.map((baseContractId) => {
```

Comment:
```suggestion
  contractNode.linearizedBaseContracts.forEach((baseContractId) => {
```

Reply:
The `.map()` methods creates a new array that is not being used, so it can use `forEach`  instead